lib/src/main/java/com/diffplug/spotless/kotlin/Version.java

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
Instead of adding a public `Version` class, use these existing methods in `KtLintStep` by making them package-private:

https://github.com/diffplug/spotless/blob/a30553386acfa24f0d97f587249c38f44feb79b6/lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java#L42-L57

Example usage:

https://github.com/diffplug/spotless/blob/a30553386acfa24f0d97f587249c38f44feb79b6/lib/src/main/java/com/diffplug/spotless/kotlin/KtLintStep.java#L112

Reply:
What is the reason to use those numbers as integers instead of using a String and then doing that under the hood?

Reply:
I moved that code to the Kotlin package, and I changed the Pattern to remove the latest dot because ktfmt doesn't use at this moment a patch in its version.